purple/settings/base.py

```diff
@@ -157,3 +157,17 @@
 CELERY_BROKER_URL = os.environ.get("PURPLE_BROKER_URL", "amqp://mq/")
 CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
 CELERY_TASK_IGNORE_RESULT = True  # ignore results unless specifically enabled
+
+# Crossref / DOI
+CROSSREF_API = "https://test.crossref.org/servlet/deposit"
+CROSSREF_USER = "user"
+CROSSREF_PASSWORD = "friend"
+CROSSREF_VERSION = "4.4.2"
```

Comment:
Will changes to `CROSSREF_VERSION` require changing the xml generation code to match the change in schema? If so, I don't think this is a setting, it's a constant in the generation code.

Reply:
Fixed with b422deac5bdc21227d7654f2d7c1465f6e479fb8